CHANGES.md

```diff
@@ -1,3 +1,25 @@
+# Version 0.7.0
+
+New features:
+* Support for encoding BigTiff ([#122](https://github.com/image-rs/image-tiff/pull/122))
```

Comment:
Needs a rebase on `master` and noting the support for new `Deflate` (https://github.com/image-rs/image-tiff/pull/132).

Reply:
Or not? we can release this as is of course :)

Reply:
Yeah. Probably is also worth pointing out the switch to `flate2`.

Reply:
Actually, since #132 gets here in a 0.7.0 and breaking changes are allowed, perhaps I should remove `InflateError` altogether now?

As part of #132 I left it but marked as deprecated only to avoid breaking API changes.

Reply:
Created PR in https://github.com/image-rs/image-tiff/pull/134.